src/Symfony/Component/Validator/Constraints/Email.php

```diff
@@ -18,13 +18,15 @@
 
 /**
  * @Annotation
+ *
  * @Target({"PROPERTY", "METHOD", "ANNOTATION"})
  *
  * @author Bernhard Schussek <bschussek@gmail.com>
  */
 #[\Attribute(\Attribute::TARGET_PROPERTY | \Attribute::TARGET_METHOD | \Attribute::IS_REPEATABLE)]
 class Email extends Constraint
 {
+    public const VALIDATION_MODE_HTML5_ALLOW_NO_TLD = 'no-tld';
```

Comment:
I would change the value of the constant. The current value makes it looks like it forbids having a TLD (which is not what it does)

Reply:
Do you prefer `allow-no-tld` ?